tests/data/end_to_end1/config.yaml

```diff
@@ -11,6 +11,7 @@ tables:
     table1:
       required: True
       metadata: tests/data/end_to_end1/meta_data/table1.json
+      expect-header: True
 
     table2:
```

Comment:
would you want to add `expect-header` to this table too? Or else add a header test

Reply:
Table 2 is `jsonl` so not applicable.